Convex_hull_d/include/CGAL/Regular_complex_d.h

```diff
@@ -86,7 +86,7 @@ class RC_vertex_d
   RC_vertex_d(Simplex_handle s, int i, const Point_d& p) :
     s_(s), index_(i), point_(p) {}
   RC_vertex_d(const Point_d& p) : index_(-42), point_(p), pp(NULL) {}
-  RC_vertex_d() :  s_(), index_(-42), pp(NULL) {}
+  RC_vertex_d() :  s_(), pp(NULL) {}
```

Comment:
Your earlier patch introduced `index_(-42)` for the other constructor, not this one. The combined diff then shows a strange move of this initialization from one line to the previous line...